lib/releaselib.sh

```diff
@@ -57,11 +57,11 @@ release::update_job_cache () {
     # the cache should be complete after that
     [[ -n ${JOB[$run]} ]] && break
```

Comment:
not related to this PR, but I don't see how this ever gets hit - `$run` is always decremented before returning here, so `$JOB[$run]` is always going to be empty.